Packs/PagerDuty/Integrations/PagerDuty/PagerDuty_test.py

```diff
@@ -428,3 +429,84 @@ def test_translate_severity(mocker, severity, expected_result):
     from PagerDuty import translate_severity
     res = translate_severity(severity)
     assert res == expected_result
+
+
+def test_paginate_with_limit(mocker: MockerFixture):
+    """Tests pagination when limit is equal to max incidents per page (100).
```

Comment:
please change the doc string to "given, when, then" format, so it will be clear(and make suer it dose) what is the input and what is the expected result, so anyone that reads it understand the case. relevant to all the tests you added.